src/sentry/integrations/client.py

```diff
@@ -119,6 +120,20 @@ def track_response_code(integration, code):
     )
 
 
+def log_request_failure(integration, code, error=None, **context):
+    logger = logging.getLogger("sentry.integrations")
+
+    logger.info(
```

Comment:
@ceorourke I think we should keep `extra` as a flat object here. Here's my suggestion:
```
extra = {
            "integration": integration,
            "status": code,
            "error": six.text_type(error[:128]) if error else None,
}
extra.update(context or {})
logger.info("integrations.http_response", extra=extra)
```